www/src/gatsby-plugin-theme-ui/index.js

```diff
@@ -298,7 +298,7 @@ const col = {
         border: darkBorder,
       },
       widget: {
-        background: darkBackground,
+        background: c.grey[90],
```

Comment:
(Sorry for async, got sidetracked) Aaaand this is where things get messy (on my end!) — we currently use `widget.background` for the feedback widget, too, which has form fields which use the same background color 🙄:
 
![image](https://user-images.githubusercontent.com/21834/66573800-88c6b100-eb73-11e9-88c6-abd111a7f9d4.png)

Until I solve that design problem … in whatever, hopefully simple, way, we can revert this change, and use `card.background` instead of `widget.background` in `src/components/layout` — messy, but gives us a white background for the showcase-modal in light mode, and the dark grey we want for dark mode.

Reply:
Thank you Dan! 🙏 
Let's merge this!

Reply:
> Oh shoot, I totally missed the feedback widget and card settings being separated in in the theme UI 🙈

And don't worry about that — lots of things to tidy up, and document — which is not your fault at all!